xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1593,8 +1593,7 @@ bool CGUIMediaWindow::OnContextButton(int itemNumber, CONTEXT_BUTTON button)
   case CONTEXT_BUTTON_BROWSE_INTO:
     {
       CFileItemPtr item = m_vecItems->Get(itemNumber);
-      if(Update(item->GetPath()))
-        return true;
+      Update(item->GetPath());
```

Comment:
I'm wondering about this one, do we only return false for unhandled messages or should it return false if Update fails?